.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+[{*.go,Makefile}]
+indent_style = tab
```

Comment:
Should this editorconfig be sent in a separate PR? how does it belong to this one?

Reply:
I could remove it, if you feel like it. It's not crucial. I just piggyback it here for simplicity.